src/components/Buttons.js

```diff
@@ -1,14 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Great job so far @christianonoh 
- Please do not disable your linters fix it.
Please check the other files and ensure you don't disable any linters checks. Linters actions help us to write clean and international standard code 

Thank You